external/lib/heap/dlmalloc/rules.mk

```diff
@@ -2,6 +2,8 @@ LOCAL_DIR := $(GET_LOCAL_DIR)
 
 MODULE := $(LOCAL_DIR)
 
+MODULE_DEFINES=MSPACES=1
```

Comment:
A comment here as to why this is set would be helpful.

Reply:
Done